docs/modules/ROOT/pages/contributors/release.adoc

```diff
@@ -0,0 +1,103 @@
+= Release Process
+
+This document explains things related to the KubeArchive release.
+
+== Commits
+
+The KubeArchive release process scans commits from the latest tag up to the current
+commit. The commits must mention the GitHub Pull Requests they come from on their
+title (first line). A commit merged from the Pull Request with ID 100 should have
+this title:
+
+[source,text]
+----
+Add feature A (#100)
+----
+
+The Pull Request mention is added by default when the commits are squashed on
+Pull Request merge.
+
+== Pull Request Labels
+
+KubeArchive follows the
+link:https://semver.org/[Semantic Versioning]
+system with a leading `v`: `vMAJOR.MINOR.PATCH`. To summarize, `MAJOR` releases
+break user-facing interfaces, `MINOR` releases introduce new features and `PATCH`
+versions introduce bugfixes or documentation changes.
+
+To decide the next version the KubeArchive release process relies on the labels
+on the GitHub Pull Requests since the latest release. The following labels impact
+which number is assigned to the next release:
```

Comment:
"To decide" feels wrong here, and I think this first sentence would be better in the active sense.

```suggestion
The KubeArchive release process relies on the labels on the GitHub Pull Requests
when determining the next version number for the release. The following labels
impact which number is updated in the next release:
```